src/config/logger.js

```diff
@@ -0,0 +1,36 @@
+const bunyan = require('bunyan');
+let RotatingFileStream = require('bunyan-rotating-file-stream');
+
+let serializer = data => {
+    let query = JSON.stringify(data.query);
+    let options = JSON.stringify(data.options || {});
+
+    return `db.${data.coll}.${data.method}(${query}, ${options});`;
+};
+
+module.exports = bunyan.createLogger({
+    name: 'StylePoint',
+    src: false,
+    streams: [
```

Comment:
The purpose of this serializer is just to format the log that will appear in the console.